spec/controllers/v2/profiles_controller_spec.rb

```diff
@@ -15,7 +15,7 @@
 
   before do
     request.headers['X-RH-IDENTITY'] = current_user.account.identity_header.raw
-    expect(controller).to receive(:rbac_allowed?).and_return(rbac_allowed?)
+    allow(controller).to receive(:rbac_allowed?).and_return(rbac_allowed?)
```

Comment:
@xmicha82 using `expect` is not recommended for mocks, you should only use it for actual testing